drivers/at86rf2xx/at86rf2xx_getset.c

```diff
@@ -138,8 +138,10 @@ uint8_t at86rf2xx_get_chan(at86rf2xx_t *dev)
 
 void at86rf2xx_set_chan(at86rf2xx_t *dev, uint8_t channel)
 {
-    if ((channel < AT86RF2XX_MIN_CHANNEL) ||
-        (channel > AT86RF2XX_MAX_CHANNEL) ||
+    if ((channel > AT86RF2XX_MAX_CHANNEL) ||
+#if AT86RF2XX_MIN_CHANNEL /* is zero for sub-GHz */
```

Comment:
Maybe duplicate the if lines for readability here, like in _netdev.c

Reply:
> Maybe duplicate the if lines for readability here, like in _netdev.c

Ping @smlng?

Reply:
well, maybe not?

Reply:
Same here, its just style.